zealot/fetcher/fetcher.ts

```diff
@@ -11,28 +11,30 @@ export type FetchArgs = {
   path: string
   method: string
   body?: string | FormData
+  headers?: Headers
   enhancers?: Enhancer[]
   signal?: AbortSignal
 }
 
 export function createFetcher(host: string) {
   return {
     async promise(args: FetchArgs) {
-      const {path, method, body, signal} = args
-      const resp = await fetch(url(host, path), {method, body, signal})
+      const {path, method, body, signal, headers} = args
+      const resp = await fetch(url(host, path), {method, body, signal, headers})
       const content = await parseContentType(resp)
       return resp.ok ? content : Promise.reject(createError(content))
     },
     async stream(args: FetchArgs): Promise<ZReponse> {
-      const {path, method, body, signal} = args
-      const resp = await fetch(url(host, path), {method, body, signal})
+      const {path, method, body, signal, headers} = args
+      const resp = await fetch(url(host, path), {method, body, signal, headers})
       if (!resp.ok) {
         const content = await parseContentType(resp)
         return Promise.reject(createError(content))
       }
       const iterator = createIterator(resp, args)
       return createStream(iterator, resp)
     },
+    // TODO: send auth in xml upload, when present
```

Comment:
Does this mean that importing a log file won't be authenticated atm?

Reply:
Shoot, yes, I didn't immediately know where to include headers and meant to circle back on this. I'll add this in, thanks for catching